index.html

```diff
@@ -184,5 +184,38 @@ <h2 class="playfairFont margTop1">About</h2>
       </div>
     </section>
   </main>
+
+  <section class="artDeco2">
+    <h2 class="playfairFont">Get Started</h2>
+
+    <p class="loremStyle">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+
+    <form action="https://formspree.io/f/xknavqkz" method="post">
+
+      <label for="name" class="label1">Your name</label>
+      <input type="text" id="name" class="fullWidth2" maxlength="30" placeholder="Full name" required>
+      <label for="email" class="label1">Your email</label>
+      <input type="email" id="email" class="fullWidth2" placeholder="Email address" required>
+      <label for="textarea" class="label1">Your message</label>
+      <textarea id="textarea" class="fullWidth3" maxlength="500" placeholder="Enter your message here..." required></textarea>
```

Comment:
- [ ] Nice one giving maxlength and placeholders to your input fields, respectively. Kindly note that to ensure that the form sends the info to formspree, it should have a name attribute.

<img width="446" alt="image" src="https://user-images.githubusercontent.com/41163571/228549921-ca0fbb73-e6d0-4078-b19d-eae8944cf9c5.png">

for example,
 `<input type="email" id="email" class="fullWidth2" placeholder="Email address" name="Email" required>`